runtime-modules/content/src/lib.rs

```diff
@@ -363,22 +363,17 @@ decl_module! {
 
             let channel_bag_id = Self::bag_id_for_channel(&channel_id);
 
-            let deletion_prize = storage::DynamicBagDeletionPrize::<T> {
-                prize: Zero::zero(), // put 0 for Giza release
-                account_id: sender,
-            };
-
             if Storage::<T>::ensure_bag_exists(&channel_bag_id).is_err() {
                 if let Some(params) = upload_params.clone() {
                     Storage::<T>::can_create_dynamic_bag_with_objects_constraints(
                         &DynamicBagIdType::<T::MemberId, T::ChannelId>::Channel(channel_id),
```

Comment:
I am not sure if it is an issue or not, but I will point it out:
Storage lead can race against `sender` (in this case) with a `update_deletion_prize` extrinsic. Changing the `deletion_prize` just before this one is executed

Reply:
It could be executed. @bedeho Should we restore commitment for the bag and object deletion prizes?

Reply:
Nicely spotted @ignazio-bovo .

Well, its not clear that this really solves the full problem in a realistic way, because all that would happen is that 99% of users using apps which just read the current deletion prize would just end up sucking up all the funds in the end anyway. So if a lead is willing to do this, he will get the funds locked none the less.

I think adding commitment is fine, but probably this should be up to council to update in the end anyway.